tests/PHPUnit/Controller/ExportDataControllerTest.php

```diff
@@ -22,12 +22,14 @@ public function testExportDataWithCart(): void
         $client->loginUser($shopUser);
 
         // go to product page
-        $client->request('GET', '/en_US/products/sport-basic-white-t-shirt');
-        $this->assertSelectorTextContains('h1', 'Sport basic white T-Shirt');
+        $client->request('GET', '/en_US/products/solar-echo-t-shirt');
+        $this->assertSelectorTextContains('h1', 'Solar Echo T-Shirt');
 
         // add product to cart
         $client->submitForm('Add to cart');
+
         $client->request('GET', '/en_US/cart/');
+
         $this->assertSelectorTextContains('h1', 'Your shopping cart');
 
         // login as admin and go to customer page
```

Comment:
got this error on rendering admin since `address.countryCode` value seems to be `anonymized-by` :thinking:
```
Uncaught PHP Exception Sylius\TwigHooks\Hookable\Renderer\Exception\HookRenderException: "An error occurred during rendering the "default_address" hook in the "sylius_admin.customer.show.content.sections.general" hookable. An exception has been thrown during the rendering of a template ("Icon "flagpack:anonymized-by" not found.")." at default_address.html.twig line 12 
```
see [default_address:12](https://github.com/Sylius/Sylius/blob/v2.0.5/src/Sylius/Bundle/AdminBundle/templates/customer/show/content/sections/general/default_address.html.twig#L12)